src/diffusers/pipelines/stable_diffusion/safety_checker.py

```diff
@@ -78,3 +78,29 @@ def forward(self, clip_input, images):
             )
 
         return images, has_nsfw_concepts
+
+    @torch.no_grad()
+    def forward_onnx(self, clip_input: torch.FloatTensor, images: torch.FloatTensor):
```

Comment:
Not sure if we should keep it separate for now, or call it from the main `forward()` when the inputs are tensors instead of numpy arrays

Reply:
Good for now, I'd say - thanks a lot for diving into this :+1:

Reply:
Where is it called from now? The new ONNX pipeline seems to invoke `__call__` as usual, doesn't it?

Reply:
I'm monkey-patching it in place of the original `forward` here: https://github.com/huggingface/diffusers/pull/399/files#diff-75394ca7b0e5dd89bbb092afbe236a44c2ad341ee791d0767422f2e68bf75c14R143
Doing the same for VAE's encoder and decoder too.

Reply:
Ah, I didn't think to look in the conversion code. Thanks!